src/xSTIR/cSTIR/include/sirf/cSTIR/stir_data_containers.h

```diff
@@ -652,6 +652,17 @@ namespace sirf {
 		}
         /// Write to file
         virtual void write(const std::string &filename) const;
+        /// Write to file using format file. Example of the nifti format is below (which in turn requires ITK).
```

Comment:
this will look rather horrible with Doxygen. Better to do something like
```doxygen
/// Write to file using format file
/*! This allows speciyfing the output file format used by STIR using a text file. 
     Keywords are specified by STIR.
    
    Example format...
   \verbatim
    OutputfileFormat Parameters:=
    ...
   \endverbatim
*/
```